src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/SeleniumTest.kt

```diff
@@ -38,20 +41,15 @@ abstract class SeleniumTest : CustomConfigTests()
         proxy.noProxy = "localhost"
         proxy.httpProxy = "localhost:" + hoverflyRule.proxyPort
         proxy.sslProxy = "localhost:" + hoverflyRule.proxyPort
-
+        System.setProperty("webdriver.chrome.whitelistedIps", "")
```

Comment:
again, just quieting the error output by stopping chrome throwing irrelevant errors trying to use ipv6: https://stackoverflow.com/questions/55844788/how-to-fix-severe-bind-failed-cannot-assign-requested-address-99-while